.github/workflows/benchmark.yml

```diff
@@ -0,0 +1,158 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+name: Run TPC-DS Benchmark
```

Comment:
This is not actually a benchmark, but correctness check (with Spark result).

Reply:
I thought about this too when reviewing this PR, but decided it is fine since we could re-purpose this workflow in future for _both_ benchmarking and correctness check. It's fine for me if we decide to change it or not.

Reply:
Okay